### ISSUE TITLE
Error authorization took too long

### ISSUE THREAD

**Issue by pgallar** [caddyserver/caddy#3955]
Hi,
I have a container instance in Azure running caddy server, but yesterday started to fail with new issue "Authorization took too long"

This is the log from muy console in azure:

{"level":"info","ts":1609915407.8514838,"msg":"using provided configuration","config_file":"/srv/app/.azure/Caddyfile","config_adapter":""}
{"level":"info","ts":1609915407.9340413,"logger":"admin","msg":"admin endpoint started","address":"tcp/localhost:2019","enforce_origin":false,"origins":["localhost:2019","[::1]:2019","127.0.0.1:2019"]}
{"level":"info","ts":1609915407.9359467,"logger":"http","msg":"server is listening only on the HTTPS port but has no TLS connection policies; adding one to enable TLS","server_name":"srv0","https_port":443}
{"level":"info","ts":1609915407.9359705,"logger":"http","msg":"enabling automatic HTTP->HTTPS redirects","server_name":"srv0"}
{"level":"info","ts":1609915407.9477515,"logger":"http","msg":"enabling automatic TLS certificate management","domains":["sugoco-srv-test.eastus.azurecontainer.io"]}
{"level":"info","ts":1609915407.9549139,"logger":"tls","msg":"cleaned up storage units"}
{"level":"info","ts":1609915407.963768,"msg":"autosaved config","file":"/root/.config/caddy/autosave.json"}
{"level":"info","ts":1609915407.9638147,"msg":"serving initial configuration"}
{"level":"info","ts":1609915407.9657798,"logger":"tls.cache.maintenance","msg":"started background certificate maintenance","cache":"0xc0002bf420"}
Successfully started Caddy (pid=28) - Caddy is running in the background
{"level":"info","ts":1609915407.9674933,"logger":"tls.obtain","msg":"acquiring lock","identifier":"sugoco-srv-test.eastus.azurecontainer.io"}
{"level":"info","ts":1609915407.9898198,"logger":"tls.obtain","msg":"lock acquired","identifier":"sugoco-srv-test.eastus.azurecontainer.io"}
{"level":"info","ts":1609915408.8800123,"logger":"tls.issuance.acme","msg":"waiting on internal rate limiter","identifiers":["sugoco-srv-test.eastus.azurecontainer.io"]}
{"level":"info","ts":1609915408.8803287,"logger":"tls.issuance.acme","msg":"done waiting on internal rate limiter","identifiers":["sugoco-srv-test.eastus.azurecontainer.io"]}
[06-Jan-2021 06:43:29] NOTICE: fpm is running, pid 36
[06-Jan-2021 06:43:29] NOTICE: ready to handle connections
{"level":"info","ts":1609915409.9376855,"logger":"tls.issuance.zerossl","msg":"generated EAB credentials","key_id":"BJFREIpalaOpk8190U9VHg"}
{"level":"info","ts":1609915411.1954863,"logger":"tls.issuance.acme","msg":"waiting on internal rate limiter","identifiers":["sugoco-srv-test.eastus.azurecontainer.io"]}
{"level":"info","ts":1609915411.1955364,"logger":"tls.issuance.acme","msg":"done waiting on internal rate limiter","identifiers":["sugoco-srv-test.eastus.azurecontainer.io"]}
{"level":"info","ts":1609915411.957841,"logger":"tls.issuance.acme.acme_client","msg":"trying to solve challenge","identifier":"sugoco-srv-test.eastus.azurecontainer.io","challenge_type":"http-01","ca":"https://acme.zerossl.com/v2/DV90"}
{"level":"error","ts":1609915713.0670962,"logger":"tls.obtain","msg":"will retry","error":"[sugoco-srv-test.eastus.azurecontainer.io] Obtain: [sugoco-srv-test.eastus.azurecontainer.io] solving challenges: [sugoco-srv-test.eastus.azurecontainer.io] authorization took too long (order=https://acme.zerossl.com/v2/DV90/order/Ghu8TMoNiU6G-Y7pvSSxQw) (ca=https://acme.zerossl.com/v2/DV90)","attempt":1,"retrying_in":60,"elapsed":305.0772431,"max_duration":2592000}
{"level":"info","ts":1609915773.616294,"logger":"tls.issuance.acme.acme_client","msg":"trying to solve challenge","identifier":"sugoco-srv-test.eastus.azurecontainer.io","challenge_type":"http-01","ca":"https://acme-staging-v02.api.letsencrypt.org/directory"}
{"level":"error","ts":1609915775.260438,"logger":"tls.issuance.acme.acme_client","msg":"challenge failed","identifier":"sugoco-srv-test.eastus.azurecontainer.io","challenge_type":"http-01","status_code":400,"problem_type":"urn:ietf:params:acme:error:connection","error":"Fetching http://sugoco-srv-test.eastus.azurecontainer.io/.well-known/acme-challenge/jEUib6F1HL3eQpFH2lj-8-d_VRiQoVwgBLCswCoxyLE: Connection reset by peer"}
{"level":"error","ts":1609915775.2605243,"logger":"tls.issuance.acme.acme_client","msg":"validating authorization","identifier":"sugoco-srv-test.eastus.azurecontainer.io","error":"authorization failed: HTTP 400 urn:ietf:params:acme:error:connection - Fetching http://sugoco-srv-test.eastus.azurecontainer.io/.well-known/acme-challenge/jEUib6F1HL3eQpFH2lj-8-d_VRiQoVwgBLCswCoxyLE: Connection reset by peer","order":"https://acme-staging-v02.api.letsencrypt.org/acme/order/17400815/215040122","attempt":1,"max_attempts":3}
{"level":"info","ts":1609915776.4356544,"logger":"tls.issuance.acme.acme_client","msg":"trying to solve challenge","identifier":"sugoco-srv-test.eastus.azurecontainer.io","challenge_type":"tls-alpn-01","ca":"https://acme-staging-v02.api.letsencrypt.org/directory"}
{"level":"info","ts":1609915776.515084,"logger":"tls","msg":"served key authentication certificate","server_name":"sugoco-srv-test.eastus.azurecontainer.io","challenge":"tls-alpn-01","remote":"10.92.0.9:61707","distributed":false}
{"level":"info","ts":1609915776.6099296,"logger":"tls","msg":"served key authentication certificate","server_name":"sugoco-srv-test.eastus.azurecontainer.io","challenge":"tls-alpn-01","remote":"10.92.0.9:61708","distributed":false}
{"level":"info","ts":1609915776.6785662,"logger":"tls","msg":"served key authentication certificate","server_name":"sugoco-srv-test.eastus.azurecontainer.io","challenge":"tls-alpn-01","remote":"10.92.0.11:63350","distributed":false}
{"level":"info","ts":1609915777.040915,"logger":"tls","msg":"served key authentication certificate","server_name":"sugoco-srv-test.eastus.azurecontainer.io","challenge":"tls-alpn-01","remote":"10.92.0.11:63351","distributed":false}
{"level":"info","ts":1609915777.1261005,"logger":"tls.issuance.acme.acme_client","msg":"validations succeeded; finalizing order","order":"https://acme-staging-v02.api.letsencrypt.org/acme/order/17400815/215040143"}
{"level":"info","ts":1609915777.584537,"logger":"tls.issuance.acme.acme_client","msg":"successfully downloaded available certificate chains","count":2,"first_url":"https://acme-staging-v02.api.letsencrypt.org/acme/cert/faf12fd0ed0d2c7b7b2778d07f9ee6fc6e4a"}
{"level":"info","ts":1609915777.5854168,"logger":"tls.issuance.acme","msg":"waiting on internal rate limiter","identifiers":["sugoco-srv-test.eastus.azurecontainer.io"]}
{"level":"info","ts":1609915777.5856876,"logger":"tls.issuance.acme","msg":"done waiting on internal rate limiter","identifiers":["sugoco-srv-test.eastus.azurecontainer.io"]}
{"level":"info","ts":1609915779.115164,"logger":"tls.issuance.acme.acme_client","msg":"trying to solve challenge","identifier":"sugoco-srv-test.eastus.azurecontainer.io","challenge_type":"http-01","ca":"https://acme.zerossl.com/v2/DV90"}
{"level":"error","ts":1609916080.421032,"logger":"tls.obtain","msg":"will retry","error":"[sugoco-srv-test.eastus.azurecontainer.io] Obtain: [sugoco-srv-test.eastus.azurecontainer.io] solving challenges: [sugoco-srv-test.eastus.azurecontainer.io] authorization took too long (order=https://acme.zerossl.com/v2/DV90/order/dLGzBN1rvPhzOCOGu5V3Jw) (ca=https://acme.zerossl.com/v2/DV90)","attempt":2,"retrying_in":120,"elapsed":672.4311811,"max_duration":2592000}
{"level":"info","ts":1609916200.8385754,"logger":"tls.issuance.acme.acme_client","msg":"validations succeeded; finalizing order","order":"https://acme-staging-v02.api.letsencrypt.org/acme/order/17400815/215042749"}
{"level":"info","ts":1609916201.295405,"logger":"tls.issuance.acme.acme_client","msg":"successfully downloaded available certificate chains","count":2,"first_url":"https://acme-staging-v02.api.letsencrypt.org/acme/cert/faebb8fe1d881241d1d21093195c3a932fd1"}
{"level":"info","ts":1609916201.2964642,"logger":"tls.issuance.acme","msg":"waiting on internal rate limiter","identifiers":["sugoco-srv-test.eastus.azurecontainer.io"]}
{"level":"info","ts":1609916201.2967117,"logger":"tls.issuance.acme","msg":"done waiting on internal rate limiter","identifiers":["sugoco-srv-test.eastus.azurecontainer.io"]}
{"level":"info","ts":1609916212.8216345,"logger":"tls.issuance.acme.acme_client","msg":"trying to solve challenge","identifier":"sugoco-srv-test.eastus.azurecontainer.io","challenge_type":"http-01","ca":"https://acme.zerossl.com/v2/DV90"}
{"level":"error","ts":1609916513.6848867,"logger":"tls.obtain","msg":"will retry","error":"[sugoco-srv-test.eastus.azurecontainer.io] Obtain: [sugoco-srv-test.eastus.azurecontainer.io] solving challenges: [sugoco-srv-test.eastus.azurecontainer.io] authorization took too long (order=https://acme.zerossl.com/v2/DV90/order/5h0XUQl_fcvrY_b5tTFwWw) (ca=https://acme.zerossl.com/v2/DV90)","attempt":3,"retrying_in":120,"elapsed":1105.6950353,"max_duration":2592000}
{"level":"info","ts":1609916634.096749,"logger":"tls.issuance.acme.acme_client","msg":"validations succeeded; finalizing order","order":"https://acme-staging-v02.api.letsencrypt.org/acme/order/17400815/215045625"}
{"level":"info","ts":1609916634.3871589,"logger":"tls.issuance.acme.acme_client","msg":"successfully downloaded available certificate chains","count":2,"first_url":"https://acme-staging-v02.api.letsencrypt.org/acme/cert/faf9dea21848e76e959a20e511f8b97b67ac"}
{"level":"info","ts":1609916634.3879712,"logger":"tls.issuance.acme","msg":"waiting on internal rate limiter","identifiers":["sugoco-srv-test.eastus.azurecontainer.io"]}
{"level":"info","ts":1609916634.3880692,"logger":"tls.issuance.acme","msg":"done waiting on internal rate limiter","identifiers":["sugoco-srv-test.eastus.azurecontainer.io"]}
{"level":"info","ts":1609916636.0806074,"logger":"tls.issuance.acme.acme_client","msg":"trying to solve challenge","identifier":"sugoco-srv-test.eastus.azurecontainer.io","challenge_type":"http-01","ca":"https://acme.zerossl.com/v2/DV90"}
{"level":"error","ts":1609916936.9911451,"logger":"tls.obtain","msg":"will retry","error":"[sugoco-srv-test.eastus.azurecontainer.io] Obtain: [sugoco-srv-test.eastus.azurecontainer.io] solving challenges: [sugoco-srv-test.eastus.azurecontainer.io] authorization took too long (order=https://acme.zerossl.com/v2/DV90/order/3-LwMJqDCi9U2mhanYOhMw) (ca=https://acme.zerossl.com/v2/DV90)","attempt":4,"retrying_in":300,"elapsed":1529.001294,"max_duration":2592000}
{"level":"info","ts":1609917237.3933747,"logger":"tls.issuance.acme.acme_client","msg":"validations succeeded; finalizing order","order":"https://acme-staging-v02.api.letsencrypt.org/acme/order/17400815/215050010"}
{"level":"info","ts":1609917237.7116227,"logger":"tls.issuance.acme.acme_client","msg":"successfully downloaded available certificate chains","count":2,"first_url":"https://acme-staging-v02.api.letsencrypt.org/acme/cert/fa6cb4020f1d74044b106089744fc19dba31"}
{"level":"info","ts":1609917237.712118,"logger":"tls.issuance.acme","msg":"waiting on internal rate limiter","identifiers":["sugoco-srv-test.eastus.azurecontainer.io"]}
{"level":"info","ts":1609917237.7121646,"logger":"tls.issuance.acme","msg":"done waiting on internal rate limiter","identifiers":["sugoco-srv-test.eastus.azurecontainer.io"]}
{"level":"info","ts":1609917239.325827,"logger":"tls.issuance.acme.acme_client","msg":"trying to solve challenge","identifier":"sugoco-srv-test.eastus.azurecontainer.io","challenge_type":"http-01","ca":"https://acme.zerossl.com/v2/DV90"}
{"level":"error","ts":1609917540.6731641,"logger":"tls.obtain","msg":"will retry","error":"[sugoco-srv-test.eastus.azurecontainer.io] Obtain: [sugoco-srv-test.eastus.azurecontainer.io] solving challenges: [sugoco-srv-test.eastus.azurecontainer.io] authorization took too long (order=https://acme.zerossl.com/v2/DV90/order/iN4zyGTcWru22f85sBoFzw) (ca=https://acme.zerossl.com/v2/DV90)","attempt":5,"retrying_in":600,"elapsed":2132.6833133,"max_duration":2592000}
{"level":"info","ts":1609918141.0749817,"logger":"tls.issuance.acme.acme_client","msg":"validations succeeded; finalizing order","order":"https://acme-staging-v02.api.letsencrypt.org/acme/order/17400815/215056412"}
{"level":"info","ts":1609918141.571084,"logger":"tls.issuance.acme.acme_client","msg":"successfully downloaded available certificate chains","count":2,"first_url":"https://acme-staging-v02.api.letsencrypt.org/acme/cert/fa5fac1b737d509f2e762fd2f31b693f09c7"}
{"level":"info","ts":1609918141.5716033,"logger":"tls.issuance.acme","msg":"waiting on internal rate limiter","identifiers":["sugoco-srv-test.eastus.azurecontainer.io"]}
{"level":"info","ts":1609918141.571648,"logger":"tls.issuance.acme","msg":"done waiting on internal rate limiter","identifiers":["sugoco-srv-test.eastus.azurecontainer.io"]}
{"level":"info","ts":1609918143.0498047,"logger":"tls.issuance.acme.acme_client","msg":"trying to solve challenge","identifier":"sugoco-srv-test.eastus.azurecontainer.io","challenge_type":"http-01","ca":"https://acme.zerossl.com/v2/DV90"}
{"level":"error","ts":1609918444.0605795,"logger":"tls.obtain","msg":"will retry","error":"[sugoco-srv-test.eastus.azurecontainer.io] Obtain: [sugoco-srv-test.eastus.azurecontainer.io] solving challenges: [sugoco-srv-test.eastus.azurecontainer.io] authorization took too long (order=https://acme.zerossl.com/v2/DV90/order/MhLhRRmh0Q8739vSzCgq_Q) (ca=https://acme.zerossl.com/v2/DV90)","attempt":6,"retrying_in":1200,"elapsed":3036.0707289,"max_duration":2592000}
{"level":"info","ts":1609919644.4731224,"logger":"tls.issuance.acme.acme_client","msg":"validations succeeded; finalizing order","order":"https://acme-staging-v02.api.letsencrypt.org/acme/order/17400815/215066595"}
{"level":"info","ts":1609919644.9367726,"logger":"tls.issuance.acme.acme_client","msg":"successfully downloaded available certificate chains","count":2,"first_url":"https://acme-staging-v02.api.letsencrypt.org/acme/cert/fa9c093698bb658a4630aa4ac6d6ad66f303"}
{"level":"info","ts":1609919644.9372005,"logger":"tls.issuance.acme","msg":"waiting on internal rate limiter","identifiers":["sugoco-srv-test.eastus.azurecontainer.io"]}
{"level":"info","ts":1609919644.937224,"logger":"tls.issuance.acme","msg":"done waiting on internal rate limiter","identifiers":["sugoco-srv-test.eastus.azurecontainer.io"]}
{"level":"info","ts":1609919646.4200122,"logger":"tls.issuance.acme.acme_client","msg":"trying to solve challenge","identifier":"sugoco-srv-test.eastus.azurecontainer.io","challenge_type":"http-01","ca":"https://acme.zerossl.com/v2/DV90"}
{"level":"error","ts":1609919947.6983418,"logger":"tls.obtain","msg":"will retry","error":"[sugoco-srv-test.eastus.azurecontainer.io] Obtain: [sugoco-srv-test.eastus.azurecontainer.io] solving challenges: [sugoco-srv-test.eastus.azurecontainer.io] authorization took too long (order=https://acme.zerossl.com/v2/DV90/order/ltPDQMoVHLRDNO-3wo-VAQ) (ca=https://acme.zerossl.com/v2/DV90)","attempt":7,"retrying_in":1200,"elapsed":4539.708491,"max_duration":2592000}
{"level":"info","ts":1609921148.109358,"logger":"tls.issuance.acme.acme_client","msg":"validations succeeded; finalizing order","order":"https://acme-staging-v02.api.letsencrypt.org/acme/order/17400815/215077993"}
{"level":"info","ts":1609921149.3062816,"logger":"tls.issuance.acme.acme_client","msg":"successfully downloaded available certificate chains","count":2,"first_url":"https://acme-staging-v02.api.letsencrypt.org/acme/cert/fa298001ca4fbd7b794f03e74e6a0726c031"}
{"level":"info","ts":1609921149.3072667,"logger":"tls.issuance.acme","msg":"waiting on internal rate limiter","identifiers":["sugoco-srv-test.eastus.azurecontainer.io"]}
{"level":"info","ts":1609921149.3073208,"logger":"tls.issuance.acme","msg":"done waiting on internal rate limiter","identifiers":["sugoco-srv-test.eastus.azurecontainer.io"]}
{"level":"info","ts":1609921150.750033,"logger":"tls.issuance.acme.acme_client","msg":"trying to solve challenge","identifier":"sugoco-srv-test.eastus.azurecontainer.io","challenge_type":"http-01","ca":"https://acme.zerossl.com/v2/DV90"}
{"level":"error","ts":1609921452.0690403,"logger":"tls.obtain","msg":"will retry","error":"[sugoco-srv-test.eastus.azurecontainer.io] Obtain: [sugoco-srv-test.eastus.azurecontainer.io] solving challenges: [sugoco-srv-test.eastus.azurecontainer.io] authorization took too long (order=https://acme.zerossl.com/v2/DV90/order/k0wwonsDUhuxI22kPLOgXg) (ca=https://acme.zerossl.com/v2/DV90)","attempt":8,"retrying_in":1800,"elapsed":6044.0791896,"max_duration":2592000}
{"level":"info","ts":1609923252.4721472,"logger":"tls.issuance.acme.acme_client","msg":"validations succeeded; finalizing order","order":"https://acme-staging-v02.api.letsencrypt.org/acme/order/17400815/215092864"}
{"level":"info","ts":1609923252.9383907,"logger":"tls.issuance.acme.acme_client","msg":"successfully downloaded available certificate chains","count":2,"first_url":"https://acme-staging-v02.api.letsencrypt.org/acme/cert/fa3dc9e6c5a5f5d0438808148eecc8375672"}
{"level":"info","ts":1609923252.9389412,"logger":"tls.issuance.acme","msg":"waiting on internal rate limiter","identifiers":["sugoco-srv-test.eastus.azurecontainer.io"]}
{"level":"info","ts":1609923252.9389932,"logger":"tls.issuance.acme","msg":"done waiting on internal rate limiter","identifiers":["sugoco-srv-test.eastus.azurecontainer.io"]}
{"level":"info","ts":1609923254.4565246,"logger":"tls.issuance.acme.acme_client","msg":"trying to solve challenge","identifier":"sugoco-srv-test.eastus.azurecontainer.io","challenge_type":"http-01","ca":"https://acme.zerossl.com/v2/DV90"}
{"level":"error","ts":1609923555.2457435,"logger":"tls.obtain","msg":"will retry","error":"[sugoco-srv-test.eastus.azurecontainer.io] Obtain: [sugoco-srv-test.eastus.azurecontainer.io] solving challenges: [sugoco-srv-test.eastus.azurecontainer.io] authorization took too long (order=https://acme.zerossl.com/v2/DV90/order/A-mLI_FUs6U2NG7tLaQmTA) (ca=https://acme.zerossl.com/v2/DV90)","attempt":9,"retrying_in":1800,"elapsed":8147.2558927,"max_duration":2592000}
{"level":"info","ts":1609925355.6405144,"logger":"tls.issuance.acme.acme_client","msg":"validations succeeded; finalizing order","order":"https://acme-staging-v02.api.letsencrypt.org/acme/order/17400815/215110129"}
{"level":"info","ts":1609925355.9359937,"logger":"tls.issuance.acme.acme_client","msg":"successfully downloaded available certificate chains","count":2,"first_url":"https://acme-staging-v02.api.letsencrypt.org/acme/cert/fa35290d852e068c23836846ea65b405c95a"}
{"level":"info","ts":1609925355.9364264,"logger":"tls.issuance.acme","msg":"waiting on internal rate limiter","identifiers":["sugoco-srv-test.eastus.azurecontainer.io"]}
{"level":"info","ts":1609925355.9365158,"logger":"tls.issuance.acme","msg":"done waiting on internal rate limiter","identifiers":["sugoco-srv-test.eastus.azurecontainer.io"]}
{"level":"info","ts":1609925357.614966,"logger":"tls.issuance.acme.acme_client","msg":"trying to solve challenge","identifier":"sugoco-srv-test.eastus.azurecontainer.io","challenge_type":"http-01","ca":"https://acme.zerossl.com/v2/DV90"}
{"level":"error","ts":1609925658.6299853,"logger":"tls.obtain","msg":"will retry","error":"[sugoco-srv-test.eastus.azurecontainer.io] Obtain: [sugoco-srv-test.eastus.azurecontainer.io] solving challenges: [sugoco-srv-test.eastus.azurecontainer.io] authorization took too long (order=https://acme.zerossl.com/v2/DV90/order/GXA8hqadAP4BkI3USY55nQ) (ca=https://acme.zerossl.com/v2/DV90)","attempt":10,"retrying_in":3600,"elapsed":10250.6401344,"max_duration":2592000}

**Comment by francislavoie** [comment]
Thanks for opening an issue! We'll look into this.

It's not immediately clear to us what is going on, so we'll need your help to understand it better.

Ideally, we need to be able to reproduce the bug _in the most minimal way possible_. This allows us to write regression tests to verify the fix is working. If we can't reproduce it, then you'll have to test our changes for us until it's fixed -- and then we can't add test cases, either.

I've attached a template below that will help make this easier and faster! It will ask for some information you've already provided; that's OK, just fill it out the best you can. :+1:

I've also included some helpful tips below the template. Feel free to let me know if you have any questions!

Thank you again for your report, we look forward to resolving it!

## Template

<pre><code>## 1. Environment

### 1a. Operating system and version

```
paste here
```


### 1b. Caddy version (run `caddy version` or paste commit SHA)

```
paste here
```


### 1c. Go version (if building Caddy from source; run `go version`)

```
paste here
```


## 2. Description

### 2a. What happens (briefly explain what is wrong)




### 2b. Why it's a bug (if it's not obvious)




### 2c. Log output

```
paste terminal output or logs here
```



### 2d. Workaround(s)




### 2e. Relevant links




## 3. Tutorial (minimal steps to reproduce the bug)




</code></pre>

## Helpful tips

1. **Environment**: Please fill out your OS and Caddy versions, even if you don't think they are relevant. (They are _always_ relevant.) If you built Caddy from source, provide the commit SHA and specify your exact Go version.

2. **Description:** Describe at a high level what the bug is. What happens? Why is it a bug? Not all bugs are obvious, so convince readers that it's actually a bug.
	- **2c) Log output:** Paste terminal output and/or complete logs in a code block. [DO NOT REDACT INFORMATION except for credentials.](https://caddy.community/t/how-to-get-help-with-caddy-more-effectively/5222)
	- **2d) Workaround:** What are you doing to work around the problem in the meantime? This can help others who encounter the same problem, until we implement a fix.
	- **2e) Relevant links:** Please link to any related issues, pull requests, docs, and/or discussion. This can add crucial context to your report.

3. **Tutorial:** What are the _minimum required specific steps_ someone needs to take in order to experience the same bug? Your goal here is to make sure that anyone else can have the same experience with the bug as you do. You are writing a tutorial, so make sure to carry it out yourself before posting it. Please:
	- Start with an empty config. Add _only_ the lines/parameters that are _absolutely required_ to reproduce the bug.
	- Do not run Caddy inside containers.
	- Run Caddy manually in your terminal; do not use systemd or other init systems.
	- If making HTTP requests, avoid web browsers. Use a simpler HTTP client instead, like `curl`.
	- Do not redact any information from your config (except credentials). Domain names are public knowledge and [often necessary for quick resolution of an issue](https://caddy.community/t/how-to-get-help-with-caddy-more-effectively/5222)!
	- Note that ignoring this advice may result in delays, or even in your issue being closed. 😞 Only actionable issues are kept open, and if there is not enough information or clarity to reproduce the bug, then the report is not actionable.

Example of a tutorial:

<blockquote>Create a config file:

```
{ ... }
```

Open terminal and run Caddy:

```
$ caddy ...
```

Make an HTTP request:

```
$ curl ...
```

Notice that the result is ___ but it should be ___.</blockquote>

**Comment by mholt** [comment]
Hmm, I can already tell this isn't a bug in Caddy.

"authorization took too long" comes from [here](https://github.com/mholt/acmez/blob/dc9c5f05ed1ecfd68f0597a7bf9ea2603433665c/acme/authorization.go#L210) when polling for an authz status change goes on for too long, usually because the CA is being slow.

The default timeout is 5 minutes. I'll ask to see if ZeroSSL can speed that up.

Going to close, since this isn't an issue with Caddy or its stack (apparently) -- but will reopen if that's not the case.

**Comment by mholt** [comment]
@pgallar So I checked with them, and I'm told that there were no validation attempts with `sugoco-srv-test.eastus.azurecontainer.io` as the domain name. Does the error still happen? If so, if you try the config locally on your own computer, does it get the same error? I expect it wouldn't work when run from your computer, of course; but I bet it's something about your Azure setup.

What's more, is the logs indicate something strange. Some log entries are not from Caddy:

```
[06-Jan-2021 06:43:29] NOTICE: fpm is running, pid 36
[06-Jan-2021 06:43:29] NOTICE: ready to handle connections
```

It also seems to be getting certs simultaneously from LE and ZeroSSL. Are you running two instances of Caddy perchance? What is your exact config?
